### ISSUE TITLE
Error messages from workflows are doubled and don't refer provide line numbers

### ISSUE THREAD

**Issue by ahpook** [lyraproj/lyra#264]
**Is your feature request related to a problem? Please describe.**
When I get errors out of a workflow, the messages are doubled up:

```
/src/lyra/local #   lyra apply azure_wordpress
2019-05-09T16:01:39.708Z [ERROR] lyra.puppet: Invoke failed: @module=Puppet error="Reference to unresolved type 'AzureRM::Resource_group' (file: /go/pkg/mo
d/github.com/lyraproj/puppet-workflow@v0.0.0-20190507102935-c5a6d8ad7529/yaml/activity.go, line: 566)" timestamp=2019-05-09T16:01:39.708Z
rpc error: code = Unknown desc = Reference to unresolved type 'AzureRM::Resource_group' (file: /go/pkg/mod/github.com/lyraproj/puppet-workflow@v0.0.0-20190
507102935-c5a6d8ad7529/yaml/activity.go, line: 566)
```

There's one with the timestamp from `lyra.puppet` then a second `rpc error` . 

Additionally, in both cases the line number/file referenced isn't the *input* file but rather the internal golang file where the error was triggered. 

**Describe the solution you'd like**
There should be only one.... 

It'd be more helpful to include the line number from the workflow itself.

**Describe alternatives you've considered**

**Comment by thallgren** [comment]
We should definitely improve the error messages to indicate what workflow and step that causes the action and include line numbers where possible. Line numbers are tricky though, since the definition is created first and then acted upon. For Puppet DSL it will be possible (we evaluate an AST that contains line numbers) but the YAML parser for instance, does not provide line numbers at all.

**Comment by thallgren** [comment]
Closed by lyraproj/wfe#28.
